web_src/less/themes/theme-arc-green.less

```diff
@@ -476,12 +484,6 @@ td.blob-excerpt {
   color: #dbdbdb;
 }
 
-.ui.active.label {
-  background: #393d4a;
-  border-color: #393d4a;
-  color: #dbdbdb;
-}
-
 .repository .ui.attached.message.isSigned.isVerified {
   background-color: #394829;
   color: var(--color-secondary-dark-6);
```

Comment:
There might be more rules to remove in `theme-arc-green.less` for the SHA labels, please verify.